doc/sphinx-guides/source/installation/config.rst

```diff
@@ -263,6 +263,153 @@ As for the "Remote only" authentication mode, it means that:
 
+
+
+Database Persistence
+--------------------
+
+The Dataverse software uses a PostgreSQL server and a Solr Search Index to store objects users create.
+You can configure basic and advanced settings of the PostgreSQL database connection with the help of
+MicroProfile Config API.
+
+Basic Database Settings
++++++++++++++++++++++++
+
+1. Any of these settings can be set via system properties (see :ref:`jvm-options`), environment variables or other
+   MicroProfile Config mechanisms supported by the appserver.
+   `See Payara docs for supported sources <https://docs.payara.fish/community/docs/documentation/microprofile/config/README.html#config-sources>`_.
+2. Remember to protect your secrets. For passwords, use an environment variable (bare minimum), a password alias named the same
+   as the key (OK) or use the "dir config source" of Payara (best).
+
+   Alias creation example:
+
+   .. code-block:: shell
+
+      echo "AS_ADMIN_ALIASPASSWORD=changeme" > /tmp/p.txt
+      asadmin create-password-alias --passwordfile /tmp/p.txt dataverse.db.password
+      rm /tmp/p.txt
+
+3. Environment variables follow the key, replacing any dot, colon, dash, etc into an underscore "_" and all uppercase
+   letters. Example: ``dataverse.db.host`` -> ``DATAVERSE_DB_HOST``
+
+.. list-table::
+   :widths: 15 60 25
+   :header-rows: 1
+   :align: left
+
+   * - MPCONFIG Key
+     - Description
+     - Default
+   * - dataverse.db.host
+     - The PostgreSQL server to connect to.
+     - ``localhost``
+   * - dataverse.db.port
+     - The PostgreSQL server port to connect to.
+     - ``5432``
+   * - dataverse.db.user
+     - The PostgreSQL user name to connect with.
+     - | ``dataverse``
+       | (installer sets to ``dvnapp``)
+   * - dataverse.db.password
+     - The PostgreSQL users password to connect with.
+
+       **Please note the safety advisory above.**
+     - *No default*
+   * - dataverse.db.name
+     - The PostgreSQL database name to use for the Dataverse installation.
+     - | ``dataverse``
+       | (installer sets to ``dvndb``)
```

Comment:
I'm not sure how to see that the default database user and database are "dataverse".

To deploy this branch I used `scripts/dev/dev-rebuild.sh` which has the names the installer uses hard coded:

DB_NAME=dvndb
DB_USER=dvnapp

I guess I could try changing that script to this...

DB_NAME=dataverse
DB_USER=dataverse

... and then deleting these lines from domain.xml:

      <system-property name="dataverse.db.user" value="dvnapp"></system-property>
      <system-property name="dataverse.db.name" value="dvndb"></system-property>

That is, without these system properties configured, it sounds like the default database user and database name will be "dataverse".

Reply:
Yes. Sorry, but for container environments it really is common practice to use the application name for the database name. As the installer sets this explicitly for any installation out there, folks should be fine.